### PR TITLE
Add sort command

### LINKED ISSUE #68: Add user command to sort by timeSlot or Deadline

### PR DESCRIPTION
**Pull request by jayhee3**
Currently, the list is not sorted by any order when the app starts. 

"sort timeslot" will sort the list by timeslot.

"sort deadline" will sort the list by deadline. 

Modules without a timeslot will be booted to the back of the list if the list is sorted by timeslot. 
Similarly, modules without a deadline will be booted to the back of the list if the list is sorted by deadline. 

All features should be working. Please help to test it out!

Closes #68 #48

### CODE REVIEW

**Review by tingyic — COMMENTED** [review]
May I ask what is the EMPTY_INPUT for?

File: src/main/java/seedu/address/Main.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
  * By having a separate main class (Main) that doesn't extend Application
  * to be the entry point of the application, we avoid this issue.
  */
+//test
```
[the remark this answers is not in the document]
Comment (jayhee3) [reply src/main/java/seedu/address/Main.java:21]: Sorry, can just ignore

**Comment by jayhee3** [comment]
> May I ask what is the EMPTY_INPUT for?

Since most of the fields are optional, when there is no input from the user for a particular field, EMPTY_INPUT is used to instantialise the field classes.

**Comment by tingyic** [comment]
> > May I ask what is the EMPTY_INPUT for?
> 
> Since most of the fields are optional, when there is no input from the user for a particular field, EMPTY_INPUT is used to instantialise the field classes.

I see, noted!

**Pull request merged**